### DISCUSSION TITLE
really vertically long element problem

### DISCUSSION THREAD

**Discussion by fo-od** [tailwindlabs/tailwindcss#12836] [Help] [answered]
I don't know how to explain this problem so I couldn't search for anything similar, but there is this problem when using h-full on an element, where it makes it really vertically long for seemingly no reason?  (I'm kind of new to using tailwind, and also a bit new to HTML and CSS too) I'm trying to make the element take up the rest of the vertical space left if there is any.
also I'm using daisyui too if that helps

please look in the CSS and HTML because I have my own things there
tailwind play link (just copy-pasted my HTML and CSS): https://play.tailwindcss.com/IuPd796FjR?size=1378x732&file=css

here is what it looks like:
[Screen recording 2024-01-25 8.42.44 PM.webm](https://github.com/tailwindlabs/tailwindcss/assets/99752044/7d912aca-f2df-4b9e-89b3-a24fb294a889)

**Comment by makibat** [comment, marked as answer]
Hey!

You could wrap your elements in a `flex` parent that has a `min-h-screen` class and make the desired child take up the rest of the space with `flex-grow: 1`, [for example](https://play.tailwindcss.com/m97Uo5hmln).

**Reply by fo-od** [reply]
thanks! i was thinking about this but wasn't really sure
